Tools/dea_tools/dask.py

```diff
@@ -41,44 +46,46 @@ def create_local_dask_cluster(spare_mem="3Gb", display_client=True, return_clien
         sys.path.append("../Scripts")
         from dea_dask import create_local_dask_cluster
 
-        create_local_dask_cluster(spare_mem='4Gb')
+        create_local_dask_cluster()
 
     Parameters
     ----------
-    spare_mem : String, optional
-        The amount of memory, in Gb, to leave for the notebook to run.
-        This memory will not be used by the cluster. e.g '3Gb'
     display_client : Bool, optional
         An optional boolean indicating whether to display a summary of
         the dask client, including a link to monitor progress of the
         analysis. Set to False to hide this display.
     return_client : Bool, optional
         An optional boolean indicating whether to return the dask client
         object.
+    n_workers : int, optional
+        Number of workers to start, default is set to 1 which works well
+        with loading ODC data.
+    threads_per_worker: int, optional
```

Comment:
This is great - can we also add a `**kwargs` here that gets passed to the `dask.distributed.Client` call at line 82? That way users can do even more custom stuff if necessary

Reply:
added kwargs (hopefully correclty)